### ISSUE TITLE
pom.xml:165-171: In the continuation of #588, all the...

### ISSUE THREAD

**Issue by 0pdd** [yegor256/cactoos#1119]
The puzzle `1082-fd48e847` from #1082 has to be resolved: 

https://github.com/yegor256/cactoos/blob/42e29c66ce3a7d82ac433e5640568cb79875db0d/pom.xml#L165-L171

The puzzle was created by rultor on 25-Apr-19. 

  

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) please, pay attention to this issue

**Comment by 0crat** [comment]
Job [#1119](https://github.com/yegor256/cactoos/issues/1119) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
The job [#1119](https://github.com/yegor256/cactoos/issues/1119) assigned to @victornoel[/z](https://www.0crat.com/u/victornoel), here is [why](https://www.0crat.com/footprint/C63314D6Z/ab5d829e-2146-4207-bba4-64dacb0329fb); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by victornoel** [comment]
@0crat waiting for #1086 to be solved, without latest cactoos-matchers, it's hell to test things like BytesOf that have side-effect when reading from streams (can read only once).

**Comment by 0crat** [comment]
> @0crat waiting for #1086 to be solved, without latest cactoos-matchers, it's hell to test things ... ([here](https://github.com/yegor256/cactoos/issues/1119#issuecomment-497306939))

@victornoel The impediment for [#1119](https://github.com/yegor256/cactoos/issues/1119) was registered successfully by @victornoel[/z](https://www.0crat.com/u/victornoel)

**Comment by victornoel** [comment]
@0crat wait for #1117 to be closed first

**Comment by 0crat** [comment]
> @0crat wait for #1117 to be closed first ([here](https://github.com/yegor256/cactoos/issues/1119#issuecomment-504671158))

@victornoel Job [#1119](https://github.com/yegor256/cactoos/issues/1119) is already on hold

**Comment by victornoel** [comment]
@0crat wait for #1158 to be merged

**Comment by 0crat** [comment]
> @0crat wait for #1158 to be merged ([here](https://github.com/yegor256/cactoos/issues/1119#issuecomment-508939281))

@victornoel Job [#1119](https://github.com/yegor256/cactoos/issues/1119) is already on hold

**Comment by 0pdd** [comment]
The puzzle `1082-fd48e847` has disappeared from the source code, that's why I closed this issue.

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1166](https://github.com/yegor256/cactoos/issues/1166) is still not solved.

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@dddff4c1-9156-49e2-a2d9-cf978c5a9655, thread: pool-11-thread-5
com.zerocracy.farm.sync.SyncProject[76] java.lang.IllegalStateException: Failed to acquire "orders.xml" in "C63314D6Z" in 2min: java.util.concurrent.locks.ReentrantLock@6d09edc9[Locked by thread pool-470-thread-1]
```

[0.48.2](https://github.com/zerocracy/farm): CID: [339a0db9-3ca5-4317-9bea-2c361145e521](https://www.0crat.com/footprint/C63314D6Z/339a0db9-3ca5-4317-9bea-2c361145e521), Type: "Close job"

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@dddff4c1-9156-49e2-a2d9-cf978c5a9655, thread: pool-11-thread-5
com.zerocracy.farm.sync.SyncProject[76] java.lang.IllegalStateException: Failed to acquire "wbs.xml" in "C63314D6Z" in 2min: java.util.concurrent.locks.ReentrantLock@6d09edc9[Locked by thread pool-470-thread-1]
```

[0.48.2](https://github.com/zerocracy/farm): CID: [339a0db9-3ca5-4317-9bea-2c361145e521](https://www.0crat.com/footprint/C63314D6Z/339a0db9-3ca5-4317-9bea-2c361145e521), Type: "Close job"

**Comment by victornoel** [comment]
@llorllale there has been some bug with 0crat (see https://github.com/zerocracy/farm/issues/2119) and 0crat didn't register the closing of the ticket, could you notify 0crat with `out` command? Thanks

**Comment by victornoel** [comment]
@0crat out

**Comment by 0crat** [comment]
@victornoel[/z](https://www.0crat.com/u/victornoel) the issue is closed not by @0pdd[/z](https://www.0crat.com/u/0pdd) (its creator); I won't close the order; please, re-open it and ask @0pdd[/z](https://www.0crat.com/u/0pdd) to close it

**Comment by victornoel** [comment]
@llorllale could you ask 0crat to `out` the ticket here too please?

**Comment by 0crat** [comment]
The job [#1119](https://github.com/yegor256/cactoos/issues/1119) is now out of scope

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @victornoel[/z](https://www.0crat.com/u/victornoel), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Quality review completed: +4 point(s) just awarded to @sereshqua[/z](https://www.0crat.com/u/sereshqua)

**Comment by 0crat** [comment]
Order was finished, quality is "good": +35 point(s) just awarded to @victornoel[/z](https://www.0crat.com/u/victornoel)

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#1222](https://github.com/yegor256/cactoos/issues/1222), [#1223](https://github.com/yegor256/cactoos/issues/1223) are still not solved; solved: [#1166](https://github.com/yegor256/cactoos/issues/1166).

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1223](https://github.com/yegor256/cactoos/issues/1223) is still not solved; solved: [#1166](https://github.com/yegor256/cactoos/issues/1166), [#1222](https://github.com/yegor256/cactoos/issues/1222).

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1434](https://github.com/yegor256/cactoos/issues/1434) is still not solved; solved: [#1166](https://github.com/yegor256/cactoos/issues/1166), [#1222](https://github.com/yegor256/cactoos/issues/1222), [#1223](https://github.com/yegor256/cactoos/issues/1223).

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1583](https://github.com/yegor256/cactoos/issues/1583) is still not solved; solved: [#1166](https://github.com/yegor256/cactoos/issues/1166), [#1222](https://github.com/yegor256/cactoos/issues/1222), [#1223](https://github.com/yegor256/cactoos/issues/1223), [#1434](https://github.com/yegor256/cactoos/issues/1434).

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1910](https://github.com/yegor256/cactoos/issues/1910) is still not solved; solved: [#1166](https://github.com/yegor256/cactoos/issues/1166), [#1222](https://github.com/yegor256/cactoos/issues/1222), [#1223](https://github.com/yegor256/cactoos/issues/1223), [#1434](https://github.com/yegor256/cactoos/issues/1434), [#1583](https://github.com/yegor256/cactoos/issues/1583).
